presto-main/src/test/java/com/facebook/presto/block/TestBlockBuilder.java

```diff
@@ -117,6 +118,9 @@ public void testNewBlockBuilderLike()
     @Test
```

Comment:
Can we use `enabled = False` here with a reason instead of throwing an error?

Reply:
When you do that the test is not marked as skipped so it looks like no tests are broken.

Reply:
Sorry, not quite bit getting the reason. You mean we still wanna run the test as if it is successful?

Reply:
I think maybe the open question is really whether this test be deleted entirely or whether someone might eventually be able to reformulate it to test the intended behavior without allocating 4GB of buffers.

IMO: Skipping the test is a little better that disabling it if we’re still holding out hope for a fix because the “skipped” mention is better than no mention- but probably not but that much ultimately. The question is- can this test case be salvaged or should we just go ahead and delete it now?

Reply:
Maybe we should just delete it. By looking at the code, it says "// We are not going to test real reservation here because allocating large amount of memory fails the Travis.". Seems it's going to be flaky by nature.

Reply:
We can, it was added in as a regression test https://github.com/prestodb/presto/blame/master/presto-main/src/test/java/com/facebook/presto/block/TestBlockBuilder.java